pages/dkp/kommander/1.4/projects/index.md

```diff
@@ -9,40 +9,44 @@ excerpt: Multi-cluster Configuration Management
+Federation, in this context, means that a central location (Kommander) hosts and pushes common configurations to all Kubernetes clusters, or a pre-defined subset group under Kommander management. That pre-defined subset group of Kubernetes clusters is called a **Project**.
 
-Projects provide central IT and business units the following benefits:
+**Projects** provide central IT and business units the following benefits:
 
-- Consistent deployment of configurations and services to clusters. 
-- Sharing of Kubernetes clusters among several teams. 
-- Access to Kubernetes Cluster Federation (KubeFed) from Kommander to coordinate the configuration of multiple Kubernetes clusters.
+- Deploying consistent configurations and services to clusters. 
+- Sharing Kubernetes clusters among several teams. 
+- Coordinating access to Kubernetes Cluster Federation (KubeFed) from Kommander for multiple clusters.
 
-Kommander allows a user to manually or dynamically (using labels) select the Kubernetes clusters associated with a Project.
-Projects support the management of configmaps, continuous deployments, secrets, services, quotas, and role-based access control by leveraging federated resources.
+Kommander allows you to manually or dynamically (using labels) select the Kubernetes clusters associated with a project.
+**Projects** support configmaps management, continuous deployments, secrets, services, quotas, and role-based access control by leveraging federated resources.
 
-## Project Namespace
+## Projects namespace
 
-A Project Namespace is the grouping of all the individual standard Kubernetes namespaces that make up the concept of a Project Namespace. Project Namespaces isolate configurations across clusters. Individual standard Kubernetes namespaces are automatically created on all clusters belonging to the project. When creating a new project, you can customize the Kubernetes namespace name that is created. A Project Namespace is a Kommander specific concept.
+A Project namespace is a Kommander specific concept that groups all the individual standard Kubernetes namespaces and creates a Project Namespace. Project Namespaces isolate configurations across clusters. Individual standard Kubernetes namespaces are automatically created on all clusters belonging to the project. When creating a new project, you can customize the Kubernetes namespace name that is created.
 
-## Create a Project
+## Create a project
 
-When you create a Project, you must specify a Project Name, a Namespace Name (optional) and a way for Kommander to determine which Kubernetes clusters are part of this project.
+Create a Project by specifying:
+- A Project Name
+- A Namespace Name (optional) 
+- A way for Kommander to determine which Kubernetes clusters are part of this project
```

Comment:
Should we be more specific and mention `labels` in the UI and a fragment of a YAML when doing it manually via the CLI ? The usage of `a way` is quite abstract to me. 

Note: I know that `a way` is inherited from the old sentence, just trying to find together a better wording.

Reply:
This is explained in more detail two paragraphs down.